src/work-allocation/containers/available-tasks/available-tasks.component.ts

```diff
@@ -80,6 +80,29 @@ export class AvailableTasksComponent extends TaskListWrapperComponent {
     });
   }
 
+  /**
+   * A User 'Claims' themselves a task and goes to the case details page for that case aka. 'Assign to me'.
+   */
+  public claimTaskAndGo(task: Task): void {
+    this.taskService.claimTask(task.id).subscribe(() => {
+      this.infoMessageCommService.nextMessage({
```

Comment:
This doesn't seem to be needed seeing as you're navigating the user away from the screen.

Reply:
Dead code, will remove